DOC-1322: RS CLI Reorg - rladmin bind page (#1916)
RedisLabs/redislabs-docs: content/rs/references/cli-utilities/rladmin/bind.md

Defines whether proxies should be included or excluded in the proxy policy for the specific database endpoint.

The include and exclude commands are overwriting commands. Multiple include or exclude commands will overwrite the proxy policy.

```sh
rladmin bind
        [ db <db:id | name> ]
        endpoint <id> { include | exclude }
        <proxy_id1 .. proxy_idN>
```

### Parameters

| Parameter | Type/Value                     | Description                                                                                   |
|-----------|--------------------------------|-----------------------------------------------------------------------------------------------|
| db        | db:\<id\><br /> name           | Only allows endpoints for the specified database                                               |
| endpoint  | endpoint ID                    | Changes proxy settings for the specified endpoint                              |
| include   |                                | Includes proxies in the proxy policy |
| exclude   |                                | Excludes proxies in the proxy policy                                                           |

> | proxy     | One or more proxy IDs          | Proxies to include or exclude                                                           |

Reviewer: ```suggestion
| proxy     | list of proxy IDs          | Proxies to include or exclude                                                           |
```

$ rladmin bind endpoint 6:1 exclude 2
Executing bind endpoint: OOO.
Finished successfully
$ rladmin status endpoints db db:6
ENDPOINTS:
DB:ID    NAME   ID                 NODE      ROLE                       SSL
db:6     tr02   endpoint:6:1       node:1    all-nodes -2               No
db:6     tr02   endpoint:6:1       node:3    all-nodes -2               No
```

## `bind endpoint <id> policy`

Changes the overall proxy policy for a specific database endpoint.

```sh
rladmin bind
        [ db <db:id | name> ]
        endpoint <id>
        policy { single | all-master-shards | all-nodes }
```

### Parameters

| Parameter | Type/Value                     | Description                                                                                   |
|-----------|--------------------------------|-----------------------------------------------------------------------------------------------|
| db        | db:\<id\><br /> name           | Only allows endpoints for the specified database                                               |
| endpoint  | endpoint ID                    | Changes proxy settings for the specified endpoint                              |

> | policy    | all-master-shards<br /> all-nodes<br /> single | Changes proxy policy to the specified policy (see below) |

Reviewer: ```suggestion
| policy    | `all-master-shards`<br /> `all-nodes`<br /> `single` | Changes proxy policy to the specified policy (see below) |
```
